### ISSUE TITLE
log-power amplitude plots?

### ISSUE THREAD

**Issue by itdaniher** [miek/inspectrum#122]
TL;DR add log-amplitude, (probably) save me having to adjust `Power max:` when I change the channel bandwidth or use the same bandwidth to look at a signal of different power.

When working with `inspectrum`, I have found interacting with data containing multiple channels of ASK signals to be trying, as the amplitude plots seem normalised by center frequency and bandwidth. This is very sensible from a DSP point of view, and like the rest of the tool is well designed in context of finding the signal in the noise.

The challenge comes as any sufficiently large filter bandwidth to show multiple channels tends to decrease the plotted amplitude of the relevant signals to barely-visible levels.

It would be useful for me to visualise the amplitude of a wide bandwidth over time, with a log-power scale, such that even the power of signals relatively spread out in frequency are visible on the same axes.

Two screenshots showing the issue at hand: same signals, almost the same `Power max:`, different bandwidths, but the amplitude plot with the greater bandwidth shows less useful information.

![screenshot from 2017-02-06 19-20-11](https://cloud.githubusercontent.com/assets/22683/22676665/aaf59bac-eca1-11e6-8098-67b0931042f0.png)

![screenshot from 2017-02-06 19-22-33](https://cloud.githubusercontent.com/assets/22683/22676680/be080bda-eca1-11e6-9e29-9c55566b5f67.png)

This issue is mirrored when changing the center frequency of a filter to look at the amplitude of signals of different power (by more than several dB). Happy to find an illustration if one would be of service.

Overall, `inspectrum` is 10/10, by and large supplanted baudline as my go-to tool for graphical signal exploration. The file I'm exploring is 11G and growing.
